provider/provider.go

```diff
@@ -2564,3 +2579,15 @@ func resourceShorelineObjectDelete(typ string, objectDef map[string]interface{})
 		return diags
 	}
 }
+
+func diffSuppressSecretValues(previousValue string, newValue string) bool {
+	previousValue = strings.ReplaceAll(previousValue, " ", "") // will have this form: ******XXXX
```

Comment:
can a secret contain whitespaces inside its value?
should this be replaced with a trim function?

Reply:
Trim sounds good.